utils/v2tofhirr4/mappings.bal

```diff
@@ -1001,6 +998,14 @@ public isolated function pv2ToEncounter(hl7v2commons:Pv2 pv2) returns internatio
         status: "in-progress"
     };
 
+    string pv211 = pv2.pv211.toString();
+    if pv211 != "" {
+        decimal|error pv211Val = decimal:fromString(pv211);
+        if pv211Val is decimal {
+            encounter.length.value = pv211Val;
+        }
```

Comment:
Should we add a warn log to log the fact that we couldn't find a mapping value?

Reply:
even for the empty case

Reply:
There are many places if we log for missing values. should we have a debug logs for those cases then?